### DISCUSSION TITLE
how to hide containers not from casaos?

### DISCUSSION THREAD

**Discussion by kuduacz** [IceWhaleTech/CasaOS#1306] [Q&A]
how to hide containers not from casaos?

**Comment by Nsbx** [comment]
Same question here.
I have installed a CI (https://woodpecker-ci.org/) app and they auto deploy some projects into a container.
At this time, the container is displayed on the front as "Legacy app"
![image](https://github.com/IceWhaleTech/CasaOS/assets/8930930/22bb90e2-f095-430a-9e19-4357a939642b)
If this is not possible, I think the easiest solution would be to prefix all casaos containers, and filter on this prefix when display it.
(I don't know GO enough to make a pull request, sorry (and there is too much Chinese comment))
